modules/nodes-cluster-resource-configure-jdk.adoc

```diff
@@ -29,20 +29,20 @@ this documentation, and may involve setting multiple additional JVM options.
 == Understanding how to override the JVM maximum heap size
 
 For many Java workloads, the JVM heap is the largest single consumer of memory.
-Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRAMFraction`)
+Currently, the OpenJDK defaults to allowing up to 1/4 (1/`-XX:MaxRamPercentage`)
 of the compute node's memory to be used for the heap, regardless of whether the
 OpenJDK is running in a container or not. It is therefore *essential* to
 override this behavior, especially if a container memory limit is also set.
 
 There are at least two ways the above can be achieved:
```

Comment:
I think you can remove the entire paragraph, and describe it as https://access.redhat.com/solutions/4886081, refer to part "How the JVM calculates the heap and default values"

Reply:
Ack.